docs/calamari/SDK.md

```diff
@@ -0,0 +1,292 @@
+# 🛠 Calamari SDK
+
+This package implements a Javascript SDK for connecting with the Manta Network. API end points to connect to Calamari:
+
+## Full Nodes
+
+- provided by Manta: `wss://ws.calamari.systems`
+- provided by third party: `wss://calamari.api.onfinality.io`
+
+## Asset API (Javascript)
+
+- KMA: `api.query.system.account(AccountId)`
+- KAR: `api.query.assets.account(8u128, AccountId)`
+- AUSD: `api.query.assets.account(9u128, AccountId)`
+- LKSM: `api.query.assets.account(10u128, AccountId)`
+- MOVR: `api.query.assets.account(11u128, AccountId)`
+- KSM: `api.query.assets.account(12u128, AccountId)`
+- PHA: `api.query.assets.account(13u128, AccountId)`
+
+## Installation
+
+```sh
+yarn install manta.js
+```
+
+### Local Development
+
+1. `git clone https://github.com/Manta-Network/sdk.git`
+2. `cd sdk/manta-js/package`
+3. `yarn`
+4. `yarn build` for use in browser or `yarn build-node` for use in node.js
+5. Add `"manta.js": "file:/<LOCAL PATH OF SDK/manta-js/package>` to your project's package.json
+6. `yarn upgrade manta.js` in your project's directory
+
+# Usage
+
+All methods are called through the `MantaPrivateWallet` class.
+
+[manta-signer](#signer) must be installed and running.
+
+> If running `manta-signer` on dev mode, you should use the following features: `features=unsafe-disable-cors,disable-restart`.
+
+Refer to [sdk/manta-js/examples](https://github.com/Manta-Network/sdk/tree/main/manta-js/examples/asset-webpack-ts#mantajs-examples) for more thorough examples, and how to run them.
+
+## Node Specific
+
+If running in node.js the wasm module assumes browser DOM exists, you must export Web API functions from node in your project as seen below.
+
+```javascript
+import fetch from 'node-fetch';
+
+// @ts-ignore
+global.fetch = fetch;
+// @ts-ignore
+global.Headers = fetch.Headers;
+// @ts-ignore
+global.Request = fetch.Request;
+// @ts-ignore
+global.Response = fetch.Response;
+```
+
+## Initialization
+
+The `Environment` flag specifies whether to connect to a local node (`ws://127.0.0.1:9944`), or the use an actual node from the specified network.
+
+The `Network` flag specifies which network to connect to, either `Dolphin`, `Calamari` or `Manta`.
+
+To switch between environments and networks, a new `MantaPrivateWallet` instance should be created.
+
+```javascript
+import { MantaPrivateWallet, Environment, Network } from 'manta.js';
+
+const prodEnvironment = sdk.Environment.Production;
+const dolphinNetwork = sdk.Network.Dolphin;
```

Comment:
use Calamari network in this file, also change example token and amount related to KMA.

Reply:
done